requirements.txt

```diff
@@ -13,3 +13,5 @@ dask[complete]>=2.6
 pyproj>=2.5
 bottleneck~=1.3.1
 python-dateutil>=2.8.1
+# This package needs to be made available on PyPI or explicitly installed
+# git+git://github.com/roocs/roocs-utils.git@master#egg=roocs_utils
```

Comment:
There is already a package on pypi. Should we use it?
https://pypi.org/project/roocs-utils/

Reply:
Yes! It would make the setup much easier. Is the version on PyPI even with the API of master?

Reply:
It would be even better if there was a conda-forge binary as well.